internal/npm_install/npm_install.bzl

```diff
@@ -24,14 +24,58 @@ See discussion in the README.
 load("//internal/node:node_labels.bzl", "get_node_label", "get_npm_label", "get_yarn_label")
 load("//internal/common:os_name.bzl", "os_name")
 
+COMMON_ATTRIBUTES = dict(dict(), **{
+    "package_json": attr.label(
+        allow_files = True,
+        mandatory = True,
+        single_file = True,
+    ),
+    "prod_only": attr.bool(
+        default = False,
+        doc = "Don't install devDependencies",
+    ),
+    "data": attr.label_list(),
+    "whitelisted_file_extensions": attr.string_list(
+        doc = """List of whitelisted file extensions.
+
+        For example, [".js", ".d.ts", ".proto", ".json", ""].
+
+        If an empty list then all file extentions are included in the package
```

Comment:
should indicate why you would want to use this option (improve performance by minimizing the number of files copied into remote/sandboxed execution)

Reply:
👍